tests/console/check_default_network_manager.pm

```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Check for network daemon
+# Maintainer: Dominik Heidler <dheidler@suse.de>
+
+use base 'consoletest';
+use strict;
+use testapi;
+use utils;
+use version_utils;
+
+sub run {
+    select_console 'root-console';
+
+    assert_script_run 'ip a';
+
+    if (is_opensuse) {
```

Comment:
could you explain why?

Reply:
systemd on SLE is compiled without networkd support.